mmv1/main.go

```diff
@@ -28,28 +28,36 @@ var version = flag.String("version", "", "optional version name. If specified, t
 var product = flag.String("product", "", "optional product name. If specified, the resources under the specific product will be generated. Otherwise, resources under all products will be generated.")
 
 // Example usage: --yaml
-var yamlMode = flag.Bool("yaml", false, "strictly copy text over from ruby yaml to go yaml")
+var yamlMode = flag.Bool("yaml", false, "copy text over from ruby yaml to go yaml")
+
+// Example usage: --template
+var templateMode = flag.Bool("template", false, "copy templates over from .erb to go .tmpl")
 
 func main() {
+
 	flag.Parse()
 
 	if *yamlMode {
 		CopyText("description:")
 		CopyText("note:")
-		return
 	}
 
-	var generateCode = true
-	var generateDocs = true
+	if *templateMode {
+		convertTemplates()
+	}
 
 	if outputPath == nil || *outputPath == "" {
-		log.Fatalf("No output path specified")
+		log.Printf("No output path specified, exiting")
+		return
 	}
 
 	if version == nil || *version == "" {
-		log.Fatalf("No version specified")
+		log.Printf("No version specified, assuming ga")
+		*version = "ga"
```

Comment:
Is beta better than ga to be assumed?

Reply:
ruby versions currently assumes ga  : https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/compiler.rb#L46

Reply:
Oh, got it.